src/Admin/SupportScreen.php

```diff
@@ -170,8 +173,8 @@ public function enqueue_assets( $hook_suffix ) {
 			];
 		}
 
-		$this->support_data->set_args( $args );
-		$data = $this->support_data->get_data();
+		$support_data = $this->injector->make( SupportData::class, [ 'args' => $args ] );
+		$data         = $support_data->get_data();
```

Comment:
Alternatively, to get the data in one line:

```suggestion
		$data = ( new SupportData( $args ) )->get_data();
```